### ISSUE TITLE
No errors generated for files with only types

### ISSUE THREAD

**Issue by sahin52** [webpack-contrib/eslint-webpack-plugin#266]
### Bug report
When I run this command:
```
npx eslint ./src/**/*.ts --config .\eslint.config.mjs
```
I get this response:
```
interfaces.ts
  1:44  error  Strings must use singlequote  @stylistic/ts/quotes
```
But when I run it with webpack plugin like this:
```
plugins: [
    new ESLintPlugin({ files: ['./src/**/*.ts'], extensions: 'ts', configType: 'flat' })
  ],
```
I don't get this error.  
The file is like this:
```
import { MyId } from "../components/providers";

export interface INotifications {
    onClick: (myId: MyId) => void;
}
```
There is a similar issue in eslint plugin: https://github.com/webpack-contrib/eslint-webpack-plugin/issues/233 it says it is completed but still doesn't work for @stylistic/ts/quotes, @stylistic/ts/semi and @stylistic/ts/member-delimiter-style.  

### Actual Behavior
eslint-webpack-plugin does not generate errors with files having only types and or other things like enum and methods that are not used.

### Expected Behavior
It should also generate errors for them.

### How Do We Reproduce?
- Create a typescript environment with the latest eslint version. 
- Create a file that only contains types. Or classes methods enums that are not being used anywhere else, for example:
```
import { MyId } from "../components/providers";
export interface INotifications {
    onClick: (myId: MyId) => void;
}
export class TestClass {
    // This should cause an error because the accessibility modifier is missing
    name: string;
    // This should cause an error because the constructor has a public modifier
    public constructor(name: string) {
        this.name = name;
    }

    public method(): void {
        this.getName();
        this.setName('');
    }

    // This should be fine because the method has an explicit accessibility modifier
    getName(): string {
        return this.name;
    }

    // This should cause an error because the setter is missing an accessibility modifier
    setName(name: string): void {
        this.name = name;
    }
}
```
Add eslint-plugin to webpack:
```
const ESLintPlugin = require('eslint-webpack-plugin');
const config = {
plugins: [
    new ESLintPlugin({ files: ['./src/**/*.ts'], extensions: 'ts', configType: 'flat' })
  ],
....
```
It does not generate error for that file.  
Both webpack and eslint is latest version

### Please paste the results of `npx webpack-cli info` here, and mention other relevant information

  System:
    OS: Windows 10 10.0.19045
    CPU: (4) x64 Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz
    Memory: 7.46 GB / 15.92 GB
  Binaries:
    Node: 20.12.2 - C:\Program Files\nodejs\node.EXE    
    npm: 10.5.0 - C:\Program Files\nodejs\npm.CMD       
  Browsers:
    Edge: Chromium (129.0.2792.65)
    Internet Explorer: 11.0.19041.4355
  Packages:
    copy-webpack-plugin: 12.0.2 => 12.0.2
    css-loader: 7.1.2 => 7.1.2
    css-minimizer-webpack-plugin: 7.0.0 => 7.0.0        
    eslint-webpack-plugin: 4.2.0 => 4.2.0
    file-loader: 6.2.0 => 6.2.0
    imports-loader: 5.0.0 => 5.0.0
    resolve-url-loader: 5.0.0 => 5.0.0
    sass-loader: 16.0.2 => 16.0.2
    stylelint-webpack-plugin: 5.0.0 => 5.0.0
    ts-loader: 9.5.1 => 9.5.1
    webpack: 5.91.0 => 5.91.0
    webpack-cli: 5.1.4 => 5.1.4
    webpack-merge-and-include-globally: 2.3.4 => 2.3.4

**Comment by alexander-akait** [comment]
Please create a reproducible repository using github, most likely you have a configuration error
